### DISCUSSION TITLE
Hack not working

### DISCUSSION THREAD

**Discussion by ewrtijetr9083j4td** [ProdigyPNP/ProdigyMathGameHacking#189] [Forum]
Whenever you go to use the hack and go on prodigy, it comes up with the error 48.
When you turn off the hack, it goes back to normal.
![Screenshot_2022-10-20_5 28_PM](https://user-images.githubusercontent.com/112988730/197071161-45dc620e-ba78-4880-9b97-7de611297c8e.png)

**Comment by afkvido** [comment]
Oh btw _.player and stuff still works. just gotta fix error 48. fuck prodigyx for being closed source
